lib/probes/v8.js

```diff
@@ -0,0 +1,139 @@
+var v8 = require('v8');
+var utils = require('../utils/module');
+
+module.exports = function (pmx) {
+  var TIME_INTERVAL = 1000;
+
+  if (typeof v8.getHeapSpaceStatistics === 'function') {
+    var heapSpaceProbes = {
+      new_space: pmx.probe().metric({
+        name: 'New space used size',
+        type: 'v8/heap/space/new',
```

Comment:
we should add the unit with a field `unit`

Reply:
Done https://github.com/keymetrics/pmx/pull/110/commits/0931cf4b8ee29393d6bb42ff23010c6dc0159d08